libc/optee/trace.c

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This **trace_printf** function looks like something new. What is it for? If it is for printing to the console, maybe we could consider reconciling it with **oe_host_printf()**?

Reply:
`trace_printf` is a function used by `libutee` which we do not include in the SDK because doing so would pull in redefinitions of `printf` and friends. In the OP-TEE world, `trace_printf` is a wrapper around `printf` to pretty-print logging information. `printf` is in turn redirected to secure serial output, if available (i.e. a serial port may be reserved to OP-TEE during boot and used for logging).